deltametrics/mask.py

```
@@ -48,51 +53,308 @@ def show(self, **kwargs):
         """
         cmap = kwargs.pop('cmap', 'gray')
         fig, ax = plt.subplots()
-        ax.imshow(self.data, cmap=cmap, **kwargs)
+        if hasattr(self, 'mask'):
+            ax.imshow(self.mask, cmap=cmap, **kwargs)
+            ax.set_title('A ' + self.mask_type + ' mask')
+        else:
```

I think I would prefer to just throw an error here, rather than plotting the input. My opinion is that it is harder to debug when you don't get what you want, but also don't get an error.

---

Swapped this for an attribute error